contrib/terraform-testing-infrastructure/aws/main.tf

```
@@ -189,9 +189,11 @@ resource "aws_instance" "accumulo-testing" {
 #
 
 locals {
-  ssh_keys   = toset(concat(var.authorized_ssh_keys, [for k in var.authorized_ssh_key_files : file(k)]))
-  manager_ip = aws_instance.accumulo-testing[0].private_ip
-  worker_ips = var.instance_count > 1 ? slice(aws_instance.accumulo-testing[*].private_ip, 1, var.instance_count) : aws_instance.accumulo-testing[0].private_ip[*]
+  ssh_keys           = toset(concat(var.authorized_ssh_keys, [for k in var.authorized_ssh_key_files : file(k)]))
+  manager_host       = aws_instance.accumulo-testing[0].public_ip
```

should this be `manager_ip` based on lines below ?

---

Yeah, good catch. It's hard when I can't even really run a plan. :)
```suggestion
  manager_ip         = aws_instance.accumulo-testing[0].public_ip
```